tutorial/developing_packages/local_package_development_flow.rst

```diff
@@ -1,4 +1,303 @@
+    │           └── deactivate_conanrun.sh
+    ├── conanfile.py
+    ├── src
+    │   ├── CMakeLists.txt
+    │   ├── CMakeUserPresets.json
+    │   ├── LICENSE
+    │   ├── README.md
+    │   ├── include
+    │   │   └── hello.h
+    │   └── src
+    │       └── hello.cpp
+    └── test_package
+        ├── CMakeLists.txt
+        ├── conanfile.py
+        └── src
+            └── example.cpp
+
+Now that all the files necessary for building are generated, you can move on to testing
+the `build()` method. 
+
+conan build
+^^^^^^^^^^^
+
+Running the After :command:`conan build` command will invoke the `build()` method:
+
+.. code-block:: python
+
+    ...
+
+    class helloRecipe(ConanFile):
+
+        ...
+    
+        def build(self):
+            cmake = CMake(self)
+            cmake.configure()
+            cmake.build()
+
+        ...
+
+Let's run :command:`conan build`:
+
+.. code-block:: bash
+
+    $ conan build .
+    ...
+    -- Conan toolchain: C++ Standard 11 with extensions ON
+    -- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
+    -- Configuring done
+    -- Generating done
+    -- Build files have been ...
+    conanfile.py (hello/1.0): CMake command: cmake --build ...
+    conanfile.py (hello/1.0): RUN: cmake --build ...
+    [100%] Built target hello
+
+For most of the recipes, the `build()` method should be very simple, and you can also
+invoke the build system directly, without invoking Conan, as you have all the necessary
+files available for building. If you check the contents of the `src` folder, you'll find a
+`CMakeUserPresets.json` file that you can use to configure and build the `conan-release`
+preset. Let's try it:
+
+.. code-block:: bash
+
+    $ cd src
+    $ cmake --preset conan-release
+    ...
+    -- Configuring done
+    -- Generating done
+
+    $ cmake --build --preset conan-release
+    ...
+    [100%] Built target hello
+
+You can check that the results of invoking CMake directly are equivalent to the ones we
+got using the :command:`conan build` command.
+
+.. include:: ../cmake_presets_note.rst
+
+conan export-pkg
+^^^^^^^^^^^^^^^^
+
+Now that we built the package binaries locally we can also package those artifacts in the
+Conan local cache using the :command:`conan export-pkg` command.
+
+..  code-block:: bash
+
+    $ conan export-pkg .
+    conanfile.py (hello/1.0) package(): Packaged 1 '.h' file: hello.h
+    conanfile.py (hello/1.0) package(): Packaged 1 '.a' file: libhello.a
+    conanfile.py (hello/1.0): Package 'b1d267f77ddd5d10d06d2ecf5a6bc433fbb7eeed' created
+    conanfile.py (hello/1.0): Created package revision f09ef573c22f3919ba26ee91ae444eaa
+    ...
+    conanfile.py (hello/1.0): Package folder /Users/...
+    conanfile.py (hello/1.0): Exported package binary
+
+Now you can list the packages in the local cache and check that the ``hello/1.0`` package
+was created.
+
+..  code-block:: bash
+
+    $ conan list hello/1.0
+    Local Cache
+    hello
+        hello/1.0
+
+
+conan test
```

Comment:
This step wouldn't be necessary now, as the above ``export-pkg`` now also runs the ``test_package`` automatically, exactly as ``conan create`` does.